apps/app_rpt/rpt_manager.c

```diff
@@ -18,8 +18,7 @@
 
 extern struct rpt rpt_vars[MAXRPTS];
 
-static char *
-ctime_no_newline(const time_t *clock)
+static char* ctime_no_newline(const time_t *clock)
```

Comment:
Actually misspoke here, sorry... what I meant is the * should be on the function name, not the char. Sorry for the mixup!

Reply:
Everyone has their coding style :-)